embulk-input-oracle/src/test/java/org/embulk/input/oracle/OracleTests.java

```diff
@@ -33,8 +32,9 @@ public static void execute(TestingEmbulk embulk, String sql) throws IOException
         String user = config.get(String.class, "user");
         String password = config.get(String.class, "password");
         String database = config.get(String.class, "database");
+
         ProcessBuilder pb = new ProcessBuilder(
-                "SQLPLUS",
+                "sql",
```

Comment:
Thank you!
I see.

But with SQLcl, some tests in `BasicTest` class failed in my environment because of file encoding.
They succeeded when I replaced
`pb.environment().put("NLS_LANG", "American_America.UTF8");`
by
`pb.environment().put("JAVA_TOOL_OPTIONS", "-Dfile.encoding=UTF8");`
in `OracleTests` class.

Would you try it in your environment?